dynawo/sources/Modeler/DataInterface/PowSyblIIDM/test/TestLineInterface.cpp

```diff
@@ -181,16 +183,10 @@ TEST(DataInterfaceTest, Line) {
   ASSERT_FALSE(li2.getInitialConnected2());
   ASSERT_EQ(li2.getR(), 0.0);
   ASSERT_EQ(li2.getX(), 0.01);
-  ASSERT_DOUBLE_EQ(li2.getP1(), 0.0);
-  ASSERT_DOUBLE_EQ(li2.getQ1(), 0.0);
-  ASSERT_DOUBLE_EQ(li2.getP2(), 0.0);
```

Comment:
Please do not remove this test that corresponds to !getInitialConnected2 and P/Q not set

Reply:
done